src/app/business/block/bucket-detail/acl/acl.component.html

```diff
@@ -0,0 +1,39 @@
+<div>
+    <div class="table-toolbar">
+        <div>Public access</div>
+        <div class="right">
+            <button  class="" pButton type="button" (click)="getAclList()" icon="fa-refresh"></button>
+        </div>
+    </div>
+    <p-tabView styleClass="ui-tabview-large">
+        <div class="content-owner">
+            
+            <p-dataTable [value]="cols">
+                <p-column [style]="{'width': '60px'}"></p-column>
+                <p-column header="Group">
+                    <ng-template pTemplate="body" let-aksk="rowData" let-i="rowIndex">
+                        <span>
+                            Everyone
+                        </span>
+                    </ng-template>
+                </p-column>
+                <p-column header="List objects">
+                    <ng-template pTemplate="body" let-aksk="rowData" let-i="rowIndex">
+                        <p-checkbox label="yes" value="read" name="choose" [(ngModel)]="checkBoxArr"></p-checkbox>
+                    </ng-template>
+                </p-column>
+                <p-column header="Write objects">
+                    <ng-template pTemplate="body" let-aksk="rowData" let-i="rowIndex">
+                        <p-checkbox  label="yes" value="write" name="choose" [(ngModel)]="checkBoxArr"></p-checkbox>
+                    </ng-template>
+                </p-column>
+                <p-column header="Operation">
+                    <ng-template pTemplate="body" let-aksk="rowData" let-i="rowIndex">
+                            <button  class="ui-button-secondary" (click)="onSubmit()" pButton type="button"  label="保存"></button>
```

Comment:
Please fix labels in Chinese to English.

Reply:
Ok, I have revised it into English